src/plots/plots.js

```diff
@@ -1564,15 +1570,20 @@ plots.extendObjectWithContainers = function(dest, src, containerPaths) {
             if(!srcContainer) continue;
 
             destProp = Lib.nestedProperty(dest, containerPaths[i]);
-
             destContainer = destProp.get();
+
             if(!Array.isArray(destContainer)) {
                 destContainer = [];
                 destProp.set(destContainer);
             }
 
             for(j = 0; j < srcContainer.length; j++) {
-                destContainer[j] = plots.extendObjectWithContainers(destContainer[j], srcContainer[j]);
+                var srcObj = srcContainer[j];
+
+                if(srcObj === null) destContainer[j] = {};
```

Comment:
> because then the user can look at the array item by item and see what happened to their input

good point here.

What if non-plain-object item were coerced to `{ visible: false }` instead? Unless you can think of a situation were http://codepen.io/etpinard/pen/ENjdBX may be useful.